### PR TITLE
FAQ and Work Process design

### PR DESCRIPTION
**Pull request by varunchinna222**

### CODE REVIEW

File: src/app/home/home.component.html [hunk 1]
```diff
@@ -95,14 +97,13 @@ <h3>Digital IT Solutions</h3>
...
+                <img src="../../assets/solutions_expert.png" alt="Thinking image" width="100%" height="600px">
             </div>
         </div>
     </div>
-
-    <section class="w-100 carousal-item animate__animated animate__zoomIn mb-5 p-5">
-        <div id="carouselExampleAutoplaying" class="carousel slide" data-bs-ride="carousel">
+    <!-- Start of carousal  -->
+    <section class="w-100 carousal-item animate__animated animate__zoomIn mb-5 p-5" style="height: 600px;">
+        <div id="carouselExampleAutoplaying" class="carousel slide" data-bs-ride="carousel" style="height: 600px;">
```
Comment (ajayreddy002) [inline src/app/home/home.component.html:106]: Remove inline style

File: src/app/home/home.component.html [hunk 2]
```diff
@@ -71,8 +71,10 @@ <h5>App Development</h5>
     <!-- End of what we do -->
 
     <!-- Start of Solutions -->
-    <h1 class="animate__animated animate__fadeInUp animate__backOutLeft solution text-center" style="font-family: Playfair Display;">Solutions</h1>
-    <div style="background-image: url('../../assets/solutions_block_bg.png'); background-size: contain; background-position: center center; background-repeat: no-repeat;">
+    <h1 class="animate__animated animate__fadeInUp animate__backOutLeft solution text-center"
```
Comment (ajayreddy002) [inline src/app/home/home.component.html:74]: Remove the dots and place like this. /assets

File: src/app/home/home.component.html [hunk 3]
```diff
@@ -122,8 +122,7 @@ <h5>Amar</h5>
                     </div>
                 </div>
                 <div class="carousel-item">
-                    <img src="../../assets/carousel_2.jpg"
-                        class="" alt="...">
+                    <img src="../../assets/carousel_2.jpg" class="" alt="...">
```
Comment (ajayreddy002) [inline src/app/home/home.component.html:125]: Same as above.

File: src/app/home/home.component.html [hunk 4]
```diff
@@ -133,14 +132,137 @@ <h5>Cabby</h5>
...
+                data-bs-slide="prev">
                 <span class="carousel-control-prev-icon" aria-hidden="true"></span>
                 <span class="visually-hidden">Next</span>
             </button>
         </div>
     </section>
+    <!-- Start of FAQ -->
+    <section class="faq-block container my-5 p-3">
+        <div class="row justify-content-evenly align-items-centre"
+            style="box-shadow: 0px 10px 60px 0px rgba(38.00000000000001, 103.00000000000009, 255, 0.1);">
```
Comment (ajayreddy002) [inline src/app/home/home.component.html:150]: Remove inline styles.

File: src/app/home/home.component.scss [hunk 5]
```diff
@@ -152,6 +152,7 @@
     font-size: 25px;
 }
 
+// carousal block
 .carousal-item {
     background-image: url("../../assets/carousel_bg.png");
```
Comment (ajayreddy002) [inline src/app/home/home.component.scss:157]: Same as above

File: src/app/home/home.component.scss [hunk 6]
```diff
@@ -204,17 +205,135 @@
...
+            color: #0b1460;
+            font-family: Playfair Display;
+            font-weight: 700;
+            font-size: 40px;
+            line-height: 50px;
+        }
+    }
+
+    .work-process-bg {
+        background-image: url(https://wordpress.zozothemes.com/briddge/wp-content/uploads/sites/2/2023/07/step-line-1-1.png);
```
Comment (ajayreddy002) [inline src/app/home/home.component.scss:288]: Download the assets

**Review by ajayreddy002 — APPROVED** [review]
Please check comments

File: src/app/home/home.component.html (hunk 1 above)
Comment (varunchinna222) [reply src/app/home/home.component.html:106]: Removed Inline styles

File: src/app/home/home.component.html (hunk 2 above)
Comment (varunchinna222) [reply src/app/home/home.component.html:74]: updated assets paths

File: src/app/home/home.component.html (hunk 4 above)
Comment (varunchinna222) [reply src/app/home/home.component.html:150]: Removed inline styles

File: src/app/home/home.component.scss (hunk 5 above)
Comment (varunchinna222) [reply src/app/home/home.component.scss:157]: Updated assets path

File: src/app/home/home.component.scss (hunk 6 above)
Comment (varunchinna222) [reply src/app/home/home.component.scss:288]: Added image lo local app

File: src/app/home/home.component.html (hunk 3 above)
Comment (varunchinna222) [reply src/app/home/home.component.html:125]: Updated assets  path

**Pull request merged**